draft-ietf-jsonpath-base.md

```diff
@@ -1192,7 +1206,7 @@ with similar XPath concepts.
 | `*`   | `*`                | wildcard: All XML elements regardless of their names                                                                                  |
 | `@`   | n/a                | attribute access: JSON values do not have attributes                                                                                  |
 | `[]`  | `[]`               | subscript operator used to iterate over XML element collections and for predicates                                                    |
-| `¦`   | `[,]`              | Union operator (results in a combination of node sets); JSONPath allows alternate names or array indices as a set                     |
+| `¦`   | `[,]`              | Union operator (results in a combination of node sets); called list operator in JSONPath, allows combining member names, array indices, and slices |
```

Comment:
... combining *quoted* member names, array indices, ...

Reply:
Is the fact that the member names are quoted relevant here?  It is not mentioned in []...

Reply:
As we explicitly distinguish between `dot-member-name` and `quoted-member-name` we should be concrete here also.